src/components/TransactionBlock/PlrStakingV2TransactionBlock.tsx

```diff
@@ -505,6 +530,9 @@ const PlrStakingV2TransactionBlock = ({
     <RouteOption
       route={availableRoutes?.find((route) => route.id === option.value)}
       isChecked={selectedRoute?.value && selectedRoute?.value === option.value}
+      cost={option.extension && `${formatAmountDisplay(option.extension, '$', 2)}`}
+      fees={option?.fees && `${formatAmountDisplay(option.fees, '$', 2)}`}
```

Comment:
`formatAmountDisplay` already returns string

```suggestion
      cost={option.extension && formatAmountDisplay(option.extension, '$', 2)}
      fees={option?.fees && formatAmountDisplay(option.fees, '$', 2)}
```